test/e2e/framework/assertions.go

```diff
@@ -191,7 +191,7 @@ func (f *Framework) GetOperatorPod(t *testing.T) *v1.Pod {
 
 	// get the operator deployment
 	operator := appsv1.Deployment{}
-	f.AssertResourceEventuallyExists("monitoring-stack-operator", "operators", &operator)(t)
+	f.AssertResourceEventuallyExists("monitoring-stack-operator-prometheus-operator", "operators", &operator)(t)
```

Comment:
```suggestion
	f.AssertResourceEventuallyExists("monitoring-stack-operator", "operators", &operator)(t)
```

Reply:
You are right! Thanks, I was running the e2e-tests in dev mode but this makes sense.